### DISCUSSION TITLE
NextJS + Express App

### DISCUSSION THREAD

**Discussion by Faizan711** [vercel/turborepo#5823] [Help]
How to setup 2 NextJS frontend applications and an Express Backend in Turborepo, I tried to do it but faced many errors, so if there was a template for this it would have been very helpful.

**Comment by anthonyshew** [comment]
You can certainly drop an Express application in after running `npx create-turbo@latest`.

Could you describe the issues you were having? Or, better, a reproduction with your best effort?

**Comment by anthonyshew** [comment]
Closing due to staleness.
